File: src/Floor.java
import java.util.ArrayList;
import java.util.List;

public class Floor {
        private int floorNumber;
    private List<Room> rooms;

    public Floor(int floorNumber) {
        this.floorNumber = floorNumber;
        this.rooms = new ArrayList<>();
            
}

    public int getFloorNumber() {
        return this.floorNumber;
    }

    public void setFloorNumber(int floorNumber) {
        this.floorNumber = floorNumber;
    }

    public List<Room> getRooms() {
        return this.rooms;
    }

    public void setRooms(List<Room> rooms) {
        this.rooms = rooms;
    }

    public void addRoom(Room room) {
        this.rooms.add(room);
    }
}

File: Python/Floor.py
class Floor:
    def __init__(self, floor_number):
        self.floor_number = floor_number
        self.zones = []

    def get_floor_number(self):
        return self.floor_number

    def set_floor_number(self, floor_number):
        self.floor_number = floor_number

    def get_zones(self):
        return self.zones

    def set_zones(self, zones):
        self.zones = zones

    def add_zone(self, zone):
        self.zones.append(zone)
        
    def remove_zone(self, zone):
        self.zones.remove(zone)
